generator/src/main/java/com/scottlogic/deg/generator/generation/GenerationConfig.java

```diff
@@ -129,6 +129,21 @@ public String toString() {
         }
     }
 
+    public enum OutputFormat {
+        CSV(Constants.OutputFormats.CSV),
+        JSON(Constants.OutputFormats.JSON);
+        private final String text;
+
+        OutputFormat(String text){
+            this.text = text;
+        }
+
+        @Override
+        public String toString() {
```

Comment:
this enum can be much simpler and remove the need to constant strings in `Constants.OutputFormats`

example:
```
    public enum OutputFormat {
        CSV, JSON
    }
    public static void main(String[] args) {
        System.out.println(OutputFormat.CSV);
        System.out.println(OutputFormat.JSON);

        OutputFormat of = OutputFormat.valueOf("CSV");
        System.out.println(of);
        of = OutputFormat.valueOf("JSON");
        System.out.println(of);
    }
```

Reply:
@mrspaceman true, this could be simplified, I was trying to maintain consistency with the other enums we have, as the _name_ and _value_ are the same it could be simplified.

Reply:
Ah, I've just remembered why this might need to remain, its so we can use this line with PicoCli. Java annotations can only include `static final` fields, not `enum` names.

```java
@CommandLine.Option(
        names = {"-o"},
        description = "Output format",
        defaultValue = GenerationConfig.Constants.OutputFormats.DEFAULT)
```